.env.local

```diff
@@ -5,4 +5,5 @@ POSTGRES_PORT=5432
 POSTGRES_DB=waect
 SECRET_KEY=waectsk
 DATABASE_URL=postgresql://$POSTGRES_USER:$POSTGRES_PASSWORD@$POSTGRES_HOST:$POSTGRES_PORT/$POSTGRES_DB
+DATABASE_URL_PROD=
```

Comment:
Are we using this for our server db at some point?

Reply:
Deleted. Eduardo suggested to create a new .env.prod file with the URL to the cloud db and add that to .gitignore which I did :)